apps/admin-gui/src/assets/i18n/en.json

```
@@ -2498,7 +2498,7 @@
     "EXT_SOURCES": {
       "TITLE": "External sources",
       "SEARCH": "Search by Id, Name or Type",
-      "LOAD_SUCCESS": "External sources successfully loaded"
+      "LOAD": "Load external sources from config file"
```

Button labels should be kept short at least imho.
```suggestion
      "LOAD": "Load configuration"
```